Sources/Bob/APIs/TravisCI/TravisCI.swift

```diff
@@ -31,6 +33,26 @@ public struct Script {
     }
 }
 
+extension Dictionary where Key: ExpressibleByStringLiteral, Value: Any {
```

Comment:
Not sure if you know, but Vapor has `JSONRepresentable`. `NodeRepresentable` could be replaced with it since here you're dealing with JSON